lib/matplotlib/gridspec.py

```diff
@@ -133,9 +136,19 @@ def get_grid_positions(self, fig):
         figLefts = [left + cellWs[2*colNum] for colNum in range(ncols)]
         figRights = [left + cellWs[2*colNum+1] for colNum in range(ncols)]
 
-
         return figBottoms, figTops, figLefts, figRights
 
+    def get_tight_rect(self):
+        'get the rectangle for use with gridspec.tight_layout'
+        return self._tight_rect
+
+    def set_tight_rect(self, tight_rect):
+        'set a rectangle for use with gridspec.tight_layout'
+        if tight_rect is not None and len(tight_rect) != 4:
+            raise ValueError('Expected keyword rect to be a list or tuple'
+                'with four elements')
+        self._tight_rect =  tight_rect
```

Comment:
too many spaces after `=`.  This must be one of the files that pep8 does not get run against.